core/src/consensus.rs

```diff
@@ -43,7 +43,7 @@ pub const REWARD: u64 = BLOCK_TIME_SEC * GRIN_BASE;
 
 /// Actual block reward for a given total fee amount
 pub fn reward(fee: u64) -> u64 {
-	REWARD + fee
+	REWARD.saturating_sub(fee)
```

Comment:
Shouldn't it be `.saturating_add(fee)` since there were an addition (`+`) operator in deleted line?

Reply:
Ouch, obviously I should sleep some more. Thanks for checking.